docs/src/main/sphinx/release/release-434.md

```diff
@@ -0,0 +1,48 @@
+# Release 434 (27 Nov 2023)
+
+## General
+
+* Add support for a `FILTER` clause to the `LISTAGG` function. ({issue}`19869`)
+* {{breaking}} Rename the `query.max-writer-tasks-count` configuration property
+  and the related `max_writer_tasks_count` session property to
+  `query.max-writer-task-count` and `max_writer_task_count`. ({issue}`19793`)
+* Fix incorrect results for queries involving comparisons between `double` and
+  `real` zero and negative zero. ({issue}`19828`)
+* Prevent table function failure when its invocation is a first query run on a
+  worker. ({issue}`19849`)
+* Prevent sudden increases in memory consumption in some queries using
+  `CROSS JOINT UNNEST`. ({issue}`19762`)
+
+## BigQuery connector
+
+* Add support for reading `json` columns. ({issue}`19790`)
+* Add support for `DELETE` statement. ({issue}`6870`)
+* Improve performance when writing rows. ({issue}`18897`)
+
+## Delta Lake connector
+
+* Improve performance when reading large checkpoint files on partitioned tables. ({issue}`19588`)
```

Comment:
@ebyhr @findepi @findinpath  .. you suggested the same text for two different but related PRs .. do we want to just link to one PR? Or have one  entry with two issue/Pr links? Or reword one entry? cc @martint @colebow

Reply:
> do we want to just link to one PR?

link to both please